streams/src/main/java/org/apache/kafka/streams/kstream/Materialized.java

```diff
@@ -34,9 +34,23 @@
 
 /**
  * Used to describe how a {@link StateStore} should be materialized.
- * You can either provide a custom {@link StateStore} backend
- * through one of the provided methods accepting a supplier or use the default RocksDB backends
- * by providing just a store name.
+ * You can either provide a custom {@link StateStore} backend through one of the provided methods accepting a supplier
+ * or use the default RocksDB backends by providing just a store name.
```

Comment:
Typo:
> or use the default RocksDB backend[] by providing ...

Reply:
I think plural is intended here, as we have "key-value", "time-windowed", and "session-windowed" RocksDB store. (Plural was there before... \cc @dguy what was your intention when doing the JavaDocs here?)